### DISCUSSION TITLE
How Nuxt.js is better than `remix.js`?

### DISCUSSION THREAD

**Discussion by codematrixaditya** [nuxt/nuxt#17249] [Questions] [answered]
I am new to web development and I am confused between nuxt.js & remix.
which should I choose?

**Comment by Flanker72** [comment]
Remix is for React, Nuxt is for Vue. It's up to you)

**Comment by TheAlexLichter** [comment, marked as answer]
I'd say it is never bad to just try out both and see what you like more! Also, as @Flanker72 mentioned, it is not only Nuxt vs. Remix but also Vue vs. React that will influence the decision ☺️

**Reply by codematrixaditya** [reply]
I love both react & vue, but react is more popular and everyone talking about it.

**Reply by codematrixaditya** [reply]
if I want to create a portfolio website which is the best React or Vue?

**Reply by Flanker72** [reply]
Both are good for that job. Choose the framework which is more convenient to you if you just starting your way.
